src/ducks/apps/index.js

```diff
@@ -456,10 +475,13 @@ export function fetchInstalledApps(lang, fetchingRegistry) {
 export function fetchRegistryApps(lang, channel = DEFAULT_CHANNEL) {
   return dispatch => {
     dispatch({ type: FETCH_APPS })
+    let filterParam = ''
+    if (CONSTANTS.filterAppType)
```

Comment:
Actually `CONSTANTS` is more a global `Config` or `Settings` object. Maybe `storeConfig` ?